File: evidence/generated-systems/golang/analytics-on-golang/analytics-on-golang/internal/model/FeatureSet.go
package model

import (
    "gorm.io/gorm"
)

//==============================================================
// FeatureSet Declaration
//==============================================================
type FeatureSet struct {
    gorm.Model
     Name                                    string
    RefreshSchedule                                                            string
    WorkspaceId         *uint
    Workspace           *AnalyticsWorkspace `gorm:"foreignKey:WorkspaceId"`
     Features           []Feature `gorm:"foreignKey:FeaturesFromFeatureSetId"`
     Datasets           []DataSet `gorm:"foreignKey:DatasetsFromFeatureSetId"`
     Models           []Model_ `gorm:"foreignKey:ModelsFromFeatureSetId"`
     ModelVersions           []ModelVersion `gorm:"foreignKey:ModelVersionsFromFeatureSetId"`
     Tags           []Tag `gorm:"foreignKey:TagsFromFeatureSetId"`
    StoreType                      FeatureStoreType

// parent associations as their child

}

File: evidence/generated-systems/rubyonrails/analytics-on-rails/analyticsonrails/app/models/featureset.rb
class FeatureSet < ApplicationRecord
  enum StoreType: [:Online, :Offline, :Hybrid]


  composed_of :cronSchedule,
    class_name: "CronSchedule",
    mapping: [
      ${$mapping}, 
      %w[cronSchedule_timezone timezone]
    ]

  has_many :Workspace, class_name: 'AnalyticsWorkspace'
  has_many :Features, class_name: 'Feature'
  has_many :Datasets, class_name: 'DataSet'
  has_many :Models, class_name: 'Model'
  has_many :ModelVersions, class_name: 'ModelVersion'
  has_many :Tags, class_name: 'Tag'

end
